packages/manager/modules/core/src/index.js

```diff
@@ -76,72 +78,74 @@ angular
   // Fix security issue: https://github.com/angular-translate/angular-translate/issues/1418
   .factory(
     'translateMissingTranslationHandler',
-    ($sanitize) => (translationId) => $sanitize(translationId),
+    /* @ngInject */ ($sanitize) => (translationId) => $sanitize(translationId),
+  )
+  .run(
+    /* @ngInject */ (tmhDynamicLocaleCache, tmhDynamicLocale, coreConfig) => {
+      const injectAngularLocale = (lang) =>
+        tmhDynamicLocaleCache.put(
+          lang,
+          angular.injector(['ngLocale']).get('$locale'),
+        );
+      const defaultLanguage = coreConfig.getUserLocale();
+      const angularLocale = kebabCase(defaultLanguage);
+
+      let angularLocalePromise;
+      switch (angularLocale) {
+        case 'de-de':
+          angularLocalePromise = import('angular-i18n/angular-locale_de-de.js');
+          break;
+        case 'en-gb':
+          angularLocalePromise = import('angular-i18n/angular-locale_en-gb.js');
+          break;
+        case 'en-us':
+          angularLocalePromise = import('angular-i18n/angular-locale_en-us.js');
+          break;
+        case 'es-es':
+          angularLocalePromise = import('angular-i18n/angular-locale_es-es.js');
+          break;
+        case 'fi-fi':
+          angularLocalePromise = import('angular-i18n/angular-locale_fi-fi.js');
+          break;
+        case 'fr-ca':
+          angularLocalePromise = import('angular-i18n/angular-locale_fr-ca.js');
+          break;
+        case 'it-it':
+          angularLocalePromise = import('angular-i18n/angular-locale_it-it.js');
+          break;
+        case 'lt-lt':
+          angularLocalePromise = import('angular-i18n/angular-locale_lt-lt.js');
+          break;
+        case 'pl-pl':
+          angularLocalePromise = import('angular-i18n/angular-locale_pl-pl.js');
+          break;
+        case 'pt-pt':
+          angularLocalePromise = import('angular-i18n/angular-locale_pt-pt.js');
+          break;
+        case 'fr-fr':
+        default:
+          angularLocalePromise = import('angular-i18n/angular-locale_fr-fr.js');
+          break;
+      }
```

Comment:
Isn't it possible to get rid of the switch and do one import with string concatenation ? Something like this : 

`
const angularLocalePromise = import('angular-i18n/angular-locale_${angularLocale}.js')
`
?

Reply:
Hello @zakariasahmane 👋 

This part was not modified in this PR and could be achieved in a specific PR. 
If manager-core was builded with `webpack` we could use `webpack. ContextReplacementPlugin ` like we did for moment (https://github.com/ovh/manager/blob/develop/packages/manager/apps/hub/webpack.config.js#L73-L76).

Here manager-core us rollup and we have `@rollup/plugin-dynamic-import-vars` in out config. Maybe it's possible to restrict the included files.